gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs/components/api-runtime-logs-quick-filters/api-runtime-logs-quick-filters.component.html

```diff
@@ -16,28 +16,38 @@
 
 -->
 <div class="quick-filters">
-  <div class="quick-filters__selectors" [formGroup]="quickFiltersForm">
+  <div class="quick-filters__selectors" *ngIf="quickFiltersForm" [formGroup]="quickFiltersForm">
```

Comment:
maybe we could use `<form>` instead of div?

Reply:
Done